content/docs/aws/authn-authz.md

```diff
@@ -0,0 +1,116 @@
++++
+title = "Authentication and Authorization"
+description = "Authentication and Authorization support for Kubeflow in AWS"
+weight = 90
++++
+
+This section shows the how to setup Kubeflow with Authentization and Authorization support in Amazon Web Services (AWS).
```

Comment:
s/Authentization and Authorization/authentication and authorization
(Fix spelling and capitalization)